design/html/favorite.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css">
+  <link rel="stylesheet" href="../css/base.css">
+  <link rel="stylesheet" href="../css/posts.css">  
+  <title>Blog</title>
+</head>
+<body>
+
+<section class="container">
+  <nav>
+    <ul class="tabs">
+      <li><a class="tab " href="posts.html">Posts</a></li>
+      <li><a class="tab" href="create.html">Create</a></li>
+      <li><a class="tab active" href="favorite.html">Favorites</a></li>
+    </ul>
+  </nav>
+
+  <div id="favorite">
+    <article>
+      <div class="panel">
```

Comment:
you don't need div here 
just `<article class="panel">`

Reply:
completed!